typing/ctype.ml

```diff
@@ -2440,7 +2455,11 @@ let constrain_type_jkind ~fixed env ty jkind =
              let recur ty's_jkinds jkinds =
                let results =
                  Misc.Stdlib.List.map3
-                   (loop ~fuel ~expanded:false ~is_open) tys ty's_jkinds jkinds
+                   (fun (ty,
+                         (* CR aspsmith: use this? *)
```

Comment:
I wasn't able to find anything that failed due to us throwing this away. Plausibly this is just fine to ignore?

Reply:
This one seems important. I think it will probably come up with tests with unboxed records.